app/models/ability.rb

```diff
@@ -240,11 +240,13 @@ def offers_package_abilities
 
   def order_abilities
     can :create, Order
-    can %i[index show update destroy transition], Order, created_by_id: @user_id
+    can %i[index show transition], Order, created_by_id: @user_id
+
     if can_manage_orders? || @api_user
-      can [:create, :index, :show, :update, :transition, :destroy, :summary], Order
```

Comment:
keep update and destroy here for those who `can_manage_orders`

Reply:
This change has been made as we discussed that no matter who the user is, if the state is cancelled they cannot update or destroy the order